Feature/descw 1261 tab 27 report

robrien-bcgov opened the pull request:
https://apps.itsm.gov.bc.ca/jira/browse/DESCW-1261

kshapka-bcgov committed:
add tab 27 report and report type picker

robrien-bcgov committed:
DESCW-1261 add model for tab 27 report

Still struggling with empty menu option pickers in reports section

robrien-bcgov committed:
adjust report selector config

robrien-bcgov committed:
fix whitespace

robrien-bcgov committed:
fix picker options and re-seed

robrien-bcgov committed:
descw-1261 finalize model structure

robrien-bcgov committed:
descw-1261 tweak the total project budget JSON output to strip the extra braces

ASpiteri-BCGov commented on backend/src/database/pick_options/dropdowns.js:
```diff
@@ -21,7 +21,7 @@ const yesNoOptions = [
 /**
- * Yes no options.
+ * Quarter options.
  */
```
something seems off, these don't look like quarters either

robrien-bcgov committed:
update query to use .where() and .select() and .join()

robrien-bcgov committed:
DESCW-1261 finish template

- made final touches to formatting
- deleted old sql testing query
- updated model to present count and totals properly
- got model to handle optional project_type without crashing (shows all project types if none provided)
- updated report selector

robrien-bcgov committed:
descw-1261 clean-up code and get Social Media selection working
